pkg/combination/combination_test.go

```diff
@@ -69,7 +90,7 @@ func TestNext(t *testing.T) {
 			for {
 				next, err := combinationStream.Next(ctx)
 				if !errors.Is(err, tt.expected.err) {
-					t.Fatal("error received while processing combination stream:", err)
+					t.Fatal("[TEST nextPreSolvedCombinations()] error received while processing combination stream:", err)
```

Comment:
nit/not blocking: Adding the identifying `[Test ...]` shouldn't be necessary here as Go should outline which test case failed.